### ISSUE TITLE
`Metrics` module not working on nodejs 7 and 8

### ISSUE THREAD

**Issue by ShockwaveNN** [ONLYOFFICE/DocumentServer#139]
1. Install latest `onlyoffice-documentserver` (4.4.1-38) using [instruction](http://helpcenter.onlyoffice.com/server/linux/document/linux-installation.aspx), but use nodejs 8 (current stable) instead of nodejs 6. 
2. `cat /var/log/onlyoffice/documentserver/metrics/err.log`
```
TypeError: The super constructor to "inherits" must not be null or undefined
    at Object.exports.inherits (util.js:961:11)
    at Object.<anonymous> (/var/www/onlyoffice/documentserver/server/Metrics/node_modules/statsd/lib/config.js:33:6)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/var/www/onlyoffice/documentserver/server/Metrics/node_modules/statsd/stats.js:4:14)
util.js:961
    throw new TypeError('The super constructor to "inherits" must not ' +
    ^
```

Same problem with nodejs 7

**Comment by konovalovsergey** [comment]
Will be fixed in the next release of the statsd
etsy/statsd#593

**Comment by trenshaw** [comment]
This still appears to be an issue for Ubuntu LTS 18.04 package nodejs/bionic,now 8.10.0~dfsg-2 amd64

**Comment by trenshaw** [comment]
I tried removing the document-server and nodejs and tried adding the NodeJS 8.x
# curl -sL https://deb.nodesource.com/setup_8.x | sudo bash -

But still encounter the same error

**Comment by trenshaw** [comment]
OK cloned from https://github.com/etsy/statsd.git and replaced statsd in /var/www/onlyoffice/documentserver/server/Metrics/node_modules .. working now

**Comment by ShockwaveNN** [comment]
@trenshaw yeah, you're right. Issiue still actual since no statsd release for 2 years, but manually cloning from git should be fine

**Comment by fluboi** [comment]
I faced the same issue on a fresh install (using debian packages). As explained before, manually cloning statsd repo does works. 
However I'm quite surprise: even with a broken Metrics module, from an end user point of view, everything was fine. What is this module useful for?

**Comment by ShockwaveNN** [comment]
Fixed in https://github.com/ONLYOFFICE/server/commit/51e8f5aa5a2a488ceee44a8f074c2e0bf805d82c and released as 5.2.0
